tests/suite/resources_utils.py

```diff
@@ -981,10 +981,10 @@ def ensure_response_from_backend(req_url, host) -> None:
     :param host:
     :return:
     """
-    for _ in range(10):
+    for _ in range(30):
         resp = requests.get(req_url, headers={"host": host}, verify=False)
         if resp.status_code != 502:
             print(f"At last after {_ * 2} seconds got 200. Continue...")
```

Comment:
that fact that you didn't get 502 doesn't mean you get 200 :)

Reply:
well, sometimes I don't expect 200, but i.e. 404 ;) I haven't ever met 504 before and still can not reproduce it in GCP or in minikube.